definitions/infra-mongodb_instance/tests/MetricRaw.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "host.hostname": "docker-desktop",
+        "metricName": "mongodb_oplog_stats_count",
+        "targetName": "docker-desktop:9126",
+        "mongodb_cluster_name": "e2e_test_cluster",
+        "newrelic.source": "metricAPI"
+    }
```

Comment:
would be ideal to have one  of each.

So we can ensure that future changes or new rules will not affect your types in all the expected prefixes.

Reply:
Test cases for all rules added